models/models.js

```diff
@@ -45,3 +46,18 @@ exports.selectCommentsByArticle = (article_id) => {
       return rows;
     });
 };
+
+exports.insertComment = (article_id, newComment) => {
+  const { body, username } = newComment;
+  const created_at = Date.now();
+  const dateObject = convertTimestampToDate({ created_at });
+  const date = dateObject.created_at;
+  return db
+    .query(
+      `INSERT INTO comments (body, author, article_id, votes, created_at) VALUES ($1, $2, $3, $4, $5) RETURNING *;`,
```

Comment:
As above, go take a look at the seed function and see how the comments table is made and populated. Specifically how it handles each column